recipes/r-metamix/meta.yaml

```
@@ -0,0 +1,93 @@
+{% set version = '0.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-metamix
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/metaMix_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/metaMix/metaMix_{{ version }}.tar.gz
+  sha256: 1818a61c459d3672c24082df523d88175042805bf7e2ed971f66971fd3e3139f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [win]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-matrix
+    - r-rmpi
+    - r-data.table >=1.9.2
+    - r-ggplot2
+    - r-gtools
```

This package requires OpenMPI. While `r-rmpi` will ensure it is installed, we should include it as a constraint, so that installations use the same version it is built against.

```suggestion
    - r-gtools
    - openmpi
```